packages/ui/src/composables/useColors.ts

```diff
@@ -88,6 +88,10 @@ export const useColors = () => {
       return prop
     }
 
+    if (isCSSVariable(prop)) {
```

Comment:
Why not:

<img width="422" alt="image" src="https://user-images.githubusercontent.com/64714442/191535906-2b94e62d-b626-45c7-a84d-11a6667c7d30.png">

Or it wa changed before?

Reply:
It's wrong test condition, not composables. 

was: [['var(--va-primary)', undefined, false], '#2C82E0'],
now: [['var(--va-info)', undefined, false], 'var(--va-info)'],

It shouldn't return variable but hex.